frontend/src/main/components/Utils/UsersTableUtils.js

```diff
@@ -0,0 +1,31 @@
+import { toast } from "react-toastify";
+
+export function cellToAxiosParamsToggleAdmin(cell){
+    return{
+        url: "/api/admin/users/toggleAdmin",
+        method: "POST",
+        params:{
+            id: cell.row.values.id
+        }
+    }
+}
+
+export function cellToAxiosParamsToggleInstructor(cell){
+    return{
+        url: "/api/admin/users/toggleInstructor",
+        method: "POST",
+        params: {
+            id: cell.row.values.id
```

Comment:
Here as well... If I'm not mistaken, this is the line that needs to be changed.

You might experiment with swagger to make sure that toggling works in swagger, then compare the URL that is being used